common/test/java/com/couchbase/lite/ReplicatorConfigurationTest.kt

```diff
@@ -71,7 +71,7 @@ class ReplicatorConfigurationTest : BaseReplicatorTest() {
         val opts = immutableConfig.connectionOptions
         assertEquals(Defaults.Replicator.HEARTBEAT, opts[C4Replicator.REPLICATOR_HEARTBEAT_INTERVAL])
         assertEquals(Defaults.Replicator.MAX_ATTEMPTS_SINGLE_SHOT - 1, opts[C4Replicator.REPLICATOR_OPTION_MAX_RETRIES])
-        assertEquals(Defaults.Replicator.MAX_ATTEMPT_WAIT_TIME, opts[C4Replicator.REPLICATOR_OPTION_MAX_RETRY_INTERVAL])
+        assertEquals(Defaults.Replicator.MAX_ATTEMPTS_WAIT_TIME, opts[C4Replicator.REPLICATOR_OPTION_MAX_RETRY_INTERVAL])
```

Comment:
To be correct, I think we will need to deprecate the old one.

Reply:
Seem like this will get taken care as a separate PR.

Reply:
Yeah.... Defaults.java is in (two) other repositories